go/ql/test/library-tests/semmle/go/frameworks/K8sIoApiCoreV1/main.go

```diff
@@ -9,86 +9,88 @@ import (
 //go:generate depstubber -vendor k8s.io/apimachinery/pkg/runtime ProtobufMarshaller,ProtobufReverseMarshaller
 
 func source() interface{} {
-	return make([]byte, 1, 1)
+	return make([]byte, 1)
```

Comment:
What's the reason for this change?

Reply:
My IDE told me it was better to do it this way. Presumably it's a built-in Go linter of some kind.